portfolio/src/main/webapp/script.js

```diff
@@ -57,6 +67,25 @@ function validateComment() {
   return true;
 }
 
+function sendComment() {
+  const name = document.getElementById("name").value;
+  const text = document.getElementById("comment").value;
+
+  if (validateComment(name, text) === true) {
+    const request = new XMLHttpRequest();
+    const id_token = user.id_token;
+    const requestData = `name=${name}&text=${text}&id_token=${id_token}`;
```

Comment:
This is not a proper way to construct urlencoded data. For example this doesn't allow the text to contain ampersands.

Reply:
I used `encodeURIComponent` to solve this problem.

Reply:
Looks great!